src/xcode/ENA/ENA/Source/Developer Menu/Features/DMWarnOthersNotificationViewController.swift

```diff
@@ -0,0 +1,198 @@
+		stackView.translatesAutoresizingMaskIntoConstraints = false
+		stackView.axis = .vertical
+		stackView.alignment = .center
+		stackView.spacing = 10
+		stackView.setCustomSpacing(20, after: descriptionLabel)
+		
+		view.addSubview(stackView)
+		NSLayoutConstraint.activate([
+			stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
+			stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
+			stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 10),
+			time1Textfield.widthAnchor.constraint(equalToConstant: 60),
+			time2Textfield.widthAnchor.constraint(equalToConstant: 60)
+		])
+		
+		// MARK: Set Default Value for the notification text filelds:
+		time1Textfield.text = "\(warnOthers.notificationOneTimer)"
+		time2Textfield.text = "\(warnOthers.notificationTwoTimer)"
+		
+		//Set the keyboard type.
+		time1Textfield.keyboardType = .numberPad
+		time2Textfield.keyboardType = .numberPad
+		self.hideKeyboardWhenTappedAround()
+		
+	}
+	
+	// MARK: Private API
+	@objc
+	private func scheduleNotificationsButtonTapped() {
+		let time1 = Int(time1Textfield.text ?? "")
+		let time2 = Int(time2Textfield.text ?? "")
+		
+		// Create an alert when user enter the wrong values.
+		if time2 ?? WarnOthersNotificationsTimer.timerOneTime.rawValue <= time1 ?? WarnOthersNotificationsTimer.timerTwoTime.rawValue {
+			
+			// Display second notification should be greater than first notification alert.
+			alertMessage(titleStr: "Please Enter the Correct Notification Time", messageStr: "Second notification time seconds should be greater than the first notification time seconds.")
+
+		} else {
+			// Save the notifications time into the SecureStore.
+			warnOthers.notificationOneTimer = TimeInterval(time1 ?? WarnOthersNotificationsTimer.timerOneTime.rawValue)
+			warnOthers.notificationTwoTimer = TimeInterval(time2 ?? WarnOthersNotificationsTimer.timerTwoTime.rawValue)
+			
+			//Display notification save alert.
+			alertMessage(titleStr: "Notifications time saved", messageStr: "Notification1 time \(time1 ?? WarnOthersNotificationsTimer.timerOneTime.rawValue) seconds & Notification2 time \(time2 ?? WarnOthersNotificationsTimer.timerTwoTime.rawValue) seconds has saved into the secure store.")
+		}
+
+	}
+	
+	
+	@objc
+	private func resetDefaultsButtonTapped() {
+		time1Textfield.text = "\(WarnOthersNotificationsTimer.timerOneTime.rawValue)"
+		time2Textfield.text = "\(WarnOthersNotificationsTimer.timerTwoTime.rawValue)"
+	}
+	
+	@objc
+	private func resetNotificationsButtonTapped() {
+		warnOthers.reset()
+		alertMessage(titleStr: "Done", messageStr: "Warn others notifications can appear again.")
+
+	}
+}
+
+// MARK: Set the alert popups.
+extension DMWarnOthersNotificationViewController {
+	func alertMessage(titleStr: String, messageStr: String) {
```

Comment:
```suggestion
	func showAlert(title: String, message: String) {
```

Without `Str` it's more swifty, and `showAlert` would be more consistent with naming in other parts of the app and makes it clearer, that this already presents the alert.

Reply:
Changed